includes/wikia/services/RevisionUpvotesService.class.php

```diff
@@ -51,7 +51,7 @@ public function removeUpvote( $id, $fromUser, $userId ) {
 
 		if ( $this->shouldStoreUserData( $userId ) ) {
 			( new \WikiaSQL() )
-				->UPDATE( self::UPVOTE_USERS_TABLE )
+				->UPDATE( static::UPVOTE_USERS_TABLE )
 				->SET_RAW( 'total', 'IF(`total` > 0, `total` - 1, 0)' )
 				->SET_RAW( 'new', 'IF(`new` > 0, `new` - 1, 0)' )
 				->WHERE( 'user_id' )->EQUAL_TO( $userId )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "user_id" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)